interfaces/webrtc-identity.idl

```diff
@@ -62,6 +66,29 @@ interface RTCIdentityAssertion {
     attribute DOMString name;
 };
 
+partial interface RTCError {
+  readonly attribute long? httpRequestStatusCode;
```

Comment:
The "Partial interface RTCError: member names are unique" subtest fails, and indeed this is already in https://w3c.github.io/webrtc-pc/#rtcerror-interface.

Reply:
Looks like https://github.com/w3c/webrtc-pc/pull/2390 missed parts of `httpRequestStatusCode` in the move. I'll file a PR.

Reply:
That was https://github.com/w3c/webrtc-pc/pull/2428. Thanks!